engines/bladerunner/font.h

```diff
@@ -23,6 +23,10 @@
 #ifndef BLADERUNNER_FONT_H
 #define BLADERUNNER_FONT_H
 
+#include "bladerunner/bladerunner.h" // needed for definition of Common::ScopedPtr (subtitles font external font file support) -- and for the subtitles relevant macro defines
```

Comment:
If this remains a compile-time option (which has already been stated that it shouldn't) the include could be moved inside the following `if` if it's not needed otherwise.

Reply:
You mean the bladerunner.h include? This was needed to be outside, since I set the SUBTITLES_SUPPORT macro in there (similar to how BLADERUNNER_DEBUG_GAME, BLADERUNNER_DEBUG_CONSOLE work there).

Reply:
Ah right, makes sense then, of course.